content/automate/ManageIQ/Transformation/TransformationHosts/ovirt_host.class/__methods__/tagasdisabled.yaml

```diff
@@ -0,0 +1,13 @@
+---
+object_type: method
+version: 1.0
+object:
+  attributes:
+    name: TagAsDisabled
+    display_name: 
+    description: 
+    scope: instance
+    language: ruby
+    location: inline
+    options: {}
```

Comment:
@fdupont-redhat Is this missing the embedded methods data since your method is using the embedded methods

Reply:
Where do you see it using an embedded method ? That one doesn't. Anyway, I see I forgot to rename the module from ConversionHost to TransformationHost.

Reply:
@fdupont-redhat in your earlier ruby code you had

```
if @debug
      ManageIQ::Automate::System::CommonMethods::MIQ_AE::Debug.new.dump_object
      ManageIQ::Automate::System::CommonMethods::MIQ_AE::Debug.new.dump_root
end
``` 
which would have needed the embedded methods, now that you have removed it, its not relevant.